types/sequelize-cursor-pagination/index.d.ts

```diff
@@ -0,0 +1,61 @@
+// Type definitions for sequelize-cursor-pagination 1.1
+// Project: https://github.com/Kaltsoon/sequelize-cursor-pagination
+// Definitions by: pilagod <https://github.com/pilagod>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/
+// TypeScript Version: 2.9
+
+import Sequelize = require('sequelize');
+
+declare namespace SequelizeCursorPagination {
+    interface Cursors {
+        before: string | null;
+        after: string | null;
+        hasNext: boolean;
+        hasPrevious: boolean;
+    }
+
+    interface BasicPaginateOptions<T> {
```

Comment:
I come up a refactoring here:
```
where?: Sequelize.FindOptions<T>['where'];
attributes?: Sequelize.FindOptions<T>['attributes'];
include?: Sequelize.FindOptions<T>['include'];
```
Is it a good practice ? Or should I do by extending & Pick:
```
interface BasicPaginateOptions<T> extends Pick<Sequelize.FindOptions<T>, 'where' | 'attributes' | 'include'> {
    ...
} 
```

Reply:
I think either should work fine. If you'd like, you can send a follow-up PR. Thanks!

Reply:
No problem, many thanks for your review 🙂